addons/account_facturx/data/facturx_templates.xml

```diff
@@ -165,7 +160,7 @@
 
                         <!-- Reference. -->
                         <ram:BuyerOrderReferencedDocument>
-                            <ram:IssuerAssignedID t-esc="'%s: %s' % (record.name, record.invoice_payment_ref) if record.invoice_payment_ref else record.name"/>
+                            <ram:IssuerAssignedID t-esc="record.invoice_payment_ref if record.invoice_payment_ref else record.name"/>
```

Comment:
Can you explain to me why you need this modif?

Reply:
Because Chorus Pro was complaining about the length of this. (Saying it's >50 char, it was not but anyway)
So I talked about it with Benj and we agreed that this change should be ok to do.

Is there a reason why it would be bad?